### PR TITLE
Notification design adjustement

### PR DESCRIPTION
**Pull request by marcelinkaaa**
Few changes to the layout, added notification_shutdown icon...

### CODE REVIEW

File: android/src/com/frostwire/android/gui/services/EngineService.java [hunk 1]
```diff
@@ -221,7 +221,7 @@ public static void updatePermanentStatusNotification(WeakReference<Context> cont
 
         // VPN status
         remoteViews.setImageViewResource(R.id.view_permanent_status_vpn_icon, isVPNactive ?
-                R.drawable.notification_vpn_on : R.drawable.notification_vpn_off);
+                R.drawable.notification_shutdown : R.drawable.notification_shutdown);
```
Comment (aldenml) [inline android/src/com/frostwire/android/gui/services/EngineService.java]: same name?

Comment (marcelinkaaa) [inline android/src/com/frostwire/android/gui/services/EngineService.java]: so Gubatron wants to change the VPN icon to shutdown icon - changing the functionality of it too. Now you will be able to shutdown the app from the drop down notification. For now - just to preview the new icon I added it there. The functionality will need to be changed by him. So just a preview for now.

Comment (gubatron) [inline android/src/com/frostwire/android/gui/services/EngineService.java]: yes, I'll jump in on this branch and add the rest now

**Comment by gubatron** [comment]
new-exit-menu-icon
new-notification-shutdown-icon

**Pull request merged**